package.json

```diff
@@ -23,9 +20,14 @@
     "test": "vitest run",
     "test:coverage": "vitest run --coverage && open ./coverage/index.html",
     "test:watch": "vitest",
+    "prep:e2e": "npm run clean && npm run build && start-server-and-test start:mocks http://localhost:3000",
+    "test:e2e": "npm run prep:e2e \"playwright test\"",
+    "test:e2e:chrome": "npm run prep:e2e \"playwright test --project=chromium\"",
+    "test:e2e:firefox": "npm run prep:e2e \"playwright test --project=firefox\"",
+    "test:e2e:webkit": "npm run prep:e2e \"playwright test --project=webkit\"",
```

Comment:
I made 2 adjustments here:
1. Prefix all e2e tests with `test:`, per our convention
2. Setup a clean, build, and start-server-and-test script with mocks to make sure that running `test:e2e` works without manually setting up the dev server.

My thought is that e2e tests should always be run against the build version anyway, definitely open to other thoughts though

Reply:
Thanks for the heads up, should all be working properly now.